minitwit/src/main/java/WebApplication.java

```diff
@@ -387,6 +396,116 @@ public static void updateLatest(Request request) {
         return "";
     };
 
+    public static Route serveSimFllws = (Request request, Response response) -> {
+
+        //update_latest(request); //TODO: call
+
+        var db = new SQLite();
+        var conn = db.getConnection();
+
+        var who_id = getUserID(db, request.params(":username"));
+
+        if (who_id == 0) {
+            conn.close();
+            response.status(404);
+            return false;
+        }
+
+        Gson gson = new Gson();
+        Fllws fllws = gson.fromJson(request.body(), Fllws.class);
+
+
+        if (Objects.equals(request.requestMethod(), "POST") && !fllws.follow.isEmpty()) {
+
+            var whom_id = getUserID(db, fllws.follow);
```

Comment:
I get a "database connection closed" on this line. Changing `db` on line 406 to `new SQLite()` fixed it. I think the getConnection function I wrote is buggy and returns closed connections if the SQLite instance has been used before...